controllers/admin/AdminShopGroupController.php

```diff
@@ -187,6 +187,15 @@ public function renderForm()
                     'name' => 'name',
                     'required' => true,
                 ],
+                [
+                    'type' => 'color',
+                    'label' => $this->trans('Color', [], 'Admin.Catalog.Feature'),
+                    'name' => 'color',
+                    'desc' => [
+                        $this->trans('It will only apply to this group of shops, each store keeps its individual color.', [], 'Admin.Shopparameters.Feature'),
```

Comment:
hi @marionf ! Does this sentence have to be part of the tooltip message or as an independent message?

Reply:
Hey @Junebyun 
This message isn't part of the tooltip, it's an independant message displayed under the field